erpnext/accounts/doctype/payment_entry/payment_entry.py

```diff
@@ -641,20 +641,18 @@ def get_orders_to_be_billed(posting_date, party_type, party,
 
 	# Add cost center condition
 	if voucher_type:
-		doc = frappe.get_doc({"doctype": voucher_type})
-		condition = ""
-		if doc and hasattr(doc, 'cost_center'):
-			condition = " and cost_center='%s'" % cost_center
```

Comment:
@Anurag810 Any specific reason for removing this?

Reply:
by mistake